scripts/compose.py

```diff
@@ -733,6 +733,30 @@ def _content(self):
         )
 
 
+class Heartbeat(BeatMixin, StackService, Service):
+    DEFAULT_COMMAND = "heartbeat -e --strict.perms=false"
+    docker_path = "beats"
+
+    def __init__(self, **options):
+        del options['enable_kibana']
```

Comment:
Nice workaround, I'll follow up on this as now multiple beats-based (apm-server & heartbeat) services could use this kind of treatment.

Reply:
"nice" is probably an overstatement, but thanks ;)